code/modules/mob/mob_defines.dm

```diff
@@ -106,6 +106,7 @@
 	var/candrop = 1
 	var/lastpuke = 0
 	var/unacidable = 0
+	var/nospells = 0 //Can't use spells. At all.
```

Comment:
The alternative to adding said mob-level var (which is useful if you want to, for any reason, prevent spells being cast) is adding an event invoked on selecting a spell
Which is both less useful and more complicated